src/controllers/expenses.js

```diff
@@ -0,0 +1,112 @@
+'use strict';
+
+const expensesService = require('../services/expenses.js');
+const usersService = require('../services/users.js');
+
+const getAll = (req, res) => {
+  const queryParams = req.query;
+
```

Comment:
You can set the headers with res.setHeader('Content-Type', 'application/json') before sending the response in some functions.